engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java

```diff
@@ -306,24 +304,6 @@ void addResponse(byte[] json, Map<String, String> properties) {
                 output.getProperties().putAll(properties);
             }
             ++count;
-            if (json[0] == '{') {
```

Comment:
This may potentially break some legacy custom handler. Maybe mark as deprecate and raise Deprecation warning for this code piece execution for now

Reply:
i'm not sure if this condition is still valid, but i'll add it back with a deprecation warning.

We removed the huggingface.parse_input and replaced it with the input formatter @sindhuvahinis implemented.

I think this is only valid if user does json encoding on the outputs from python side that are received on java side